packages/core/src/external/cda/__tests__/partition-payload.test.ts

```diff
@@ -0,0 +1,85 @@
+const MOCK_MAX_CHUNK_SIZE = 100_000;
+
+// Mock the module before imports
+jest.mock("../partition-payload", () => {
+  const actual = jest.requireActual("../partition-payload");
+  const mockPartitionPayload = function (payloadRaw: string): string[] {
+    const originalMaxSize = actual.MAX_CHUNK_SIZE_IN_BYTES;
+    actual.MAX_CHUNK_SIZE_IN_BYTES = MOCK_MAX_CHUNK_SIZE;
+    const result = actual.partitionPayload(payloadRaw);
+    actual.MAX_CHUNK_SIZE_IN_BYTES = originalMaxSize;
+    return result;
+  };
+
+  return {
+    __esModule: true,
+    ...actual,
+    MAX_CHUNK_SIZE_IN_BYTES: MOCK_MAX_CHUNK_SIZE,
+    partitionPayload: mockPartitionPayload,
+  };
+});
```

Comment:
🧙 

To simplify these we could pass the maxChunkSize as parameter, having `MAX_CHUNK_SIZE_IN_BYTES` as the default value. This way, tests can easily adjust the size without resorting to these wizardries... 😄

Reply:
yes, good point.. i was thinking to avoid using that as a param exclusively for the tests, but it does make em so much cleaver